ERC20_LIST.json

```diff
@@ -8,7 +8,7 @@
         "useInSwap": true,
         "hasPermit": true,
         "permitVersion": "1",
-        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-token-list/main/src/assets/tokens/agEUR.svg"
+        "logoURI": "https://raw.githubusercontent.com/AngleProtocol/angle-assets/main/0_tokens/agEUR/agEUR-new1.svg"
```

Comment:
so the goal is to no longer have assets in this repo?

Reply:
Like there's no reason to have them in this repo in the end